src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -644,12 +654,19 @@ class AddItemsPage extends Component {
       buttons: [
         {
           label: this.props.translate('react.default.yes.label', 'Yes'),
-          onClick: onConfirm,
+          onClick: () => {
+            this.didUserConfirmAlert = true;
+          },
         },
         {
           label: this.props.translate('react.default.no.label', 'No'),
         },
       ],
+      afterClose: () => {
+        if (this.didUserConfirmAlert) {
```

Comment:
What will happen if the user confirms the first one, then not confirms the second one, and then retriggers again both modals and wants to close the first one, will it trigger the second one either way because `didUserConfirmAlert` is not reset to the initial `false` state?

Reply:
You're right, I didn't think about this case